calcOutdatedMetrics.js

```diff
@@ -1,69 +1,80 @@
+const { exec } = require("child_process");
```

Comment:
huh, so this is an internal node package that's available after the github action setup?
Is there a way to pin the node version in the action yml?

Reply:
if it's internal then it's controlled by the node version

Reply:
where's the node version defined though?

Reply:
*updates to `node 19`, bricks all the code
lol